xwiki-platform-core/xwiki-platform-index/xwiki-platform-index-test/xwiki-platform-index-test-docker/src/test/it/org/xwiki/index/test/ui/docker/DeletedDocsIT.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package org.xwiki.index.test.ui.docker;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Optional;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.xwiki.index.test.po.DeletedDocsPage;
+import org.xwiki.index.test.po.RestoreDocumentConfirmationModal;
+import org.xwiki.test.docker.junit5.TestReference;
+import org.xwiki.test.docker.junit5.UITest;
+import org.xwiki.test.ui.TestUtils;
+import org.xwiki.test.ui.po.DeletePageConfirmationPage;
+import org.xwiki.test.ui.po.DeletingPage;
+import org.xwiki.test.ui.po.ViewPage;
+
+/**
+ * Tests for Deleted Pages page.
+ *
+ * @version $Id$
+ * @since 14.4RC1
+ */
+@UITest
+public class DeletedDocsIT
+{
+    @BeforeEach
+    void setUp(TestUtils setup)
+    {
+        setup.loginAsSuperAdmin();
+    }
+
+    @Test
+    @Order(1)
+    void restoreAfterDocumentWasReacreated(TestUtils testUtils, TestReference reference) throws Exception
```

Comment:
typo

Reply:
I don't understand the name of this test as I don't see what "restore after document was recreated" means, and how it's related to deleting.

Reply:
Maybe `restoreDocumentAfterDeletionAndRecreation`? (if that's what it does)

Reply:
Yes, that's the scope of the test. Updating it